### PR TITLE
design for tablet

### PR DESCRIPTION
**Pull request by ManhNTX**
### Issues

- https://github.com/linagora/tmail-flutter/issues/1034

### Resolved

https://user-images.githubusercontent.com/107173849/195243885-ca414eda-cdb9-48a8-9525-cb86e83f011b.mov

Selected color:

https://user-images.githubusercontent.com/107173849/195299468-e4d7600b-8553-44e8-8074-3cbee26a2aad.mov

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

File: lib/richtext_controller.dart [hunk 1]
```diff
@@ -13,13 +16,23 @@ class RichTextController {
...
   final selectedTextColor = ValueNotifier<ui.Color>(ui.Colors.black);
   final selectedTextBackgroundColor = ValueNotifier<ui.Color>(ui.Colors.white);
-  final headerStyleTypeApply = ValueNotifier<HeaderStyleType>(HeaderStyleType.normal);
-  final StreamController<bool> richTextStreamController = StreamController<bool>.broadcast();
+  final headerStyleTypeApply =
+      ValueNotifier<HeaderStyleType>(HeaderStyleType.normal);
+  final dxRichTextButtonPosition = ValueNotifier<int>(35);
+  final currentIndexStackOverlayRichTextForTablet = ValueNotifier<int>(0);
+
+
```
Comment (dab246) [inline lib/richtext_controller.dart:31]: remove line

File: lib/richtext_controller.dart [hunk 2]
```diff
@@ -35,77 +48,140 @@ class RichTextController {
...
+            Set.from(listSpecialTextStyleApply.value)
+              ..remove(SpecialStyleType.strikeThrough);
         isStrikeThroughAppended = false;
       }
     };
   }
 
-  void selectTextStyleType(SpecialStyleType richTextStyleType) {
+  void selectTextStyleType(
+      SpecialStyleType richTextStyleType, BuildContext context) {
```
Comment (dab246) [inline lib/richtext_controller.dart:100]: Each parameter should be in 1 line if there are too many parameters

File: lib/richtext_controller.dart [hunk 3]
```diff
@@ -35,77 +48,140 @@ class RichTextController {
...
+          ),
+        ),
+      ),
+    );
+
+    showDialog(
+        context: context,
+        builder: (BuildContext context) {
+          return dialog;
+        });
```
Comment (dab246) [inline lib/richtext_controller.dart:184]: remove 2 indent

**Review by dab246 — COMMENTED** [review]
**Avoid code formatting**

**Review by dab246 — APPROVED** [review]

**Pull request merged**